eclair-core/src/main/scala/fr/acinq/eclair/payment/PaymentRequest.scala

```diff
@@ -200,20 +200,45 @@ object PaymentRequest {
   }
 
   /**
-    * Routing Info Tag
+    * Hidden hop
     *
     * @param pubkey          node id
     * @param channelId       channel id
     * @param fee             node fee
     * @param cltvExpiryDelta node cltv expiry delta
     */
-  case class RoutingInfoTag(pubkey: PublicKey, channelId: BinaryData, fee: Long, cltvExpiryDelta: Int) extends Tag {
+  case class HiddenHop(pubkey: PublicKey, channelId: BinaryData, fee: Long, cltvExpiryDelta: Int) {
```

Comment:
nitpick: why not Hop  instead of HiddenHop ? They don't have to be private and it sounds a bit ominous

Reply:
to avoid a possible confusion with other Hop: `case class Hop(nodeId: PublicKey, nextNodeId: PublicKey, lastUpdate: ChannelUpdate)`. I'll change this one to Hop too if this is not a problem.